pyproject.toml

```diff
@@ -26,7 +26,7 @@ dependencies = [
     "filetype~=1.2.0",
     "gql==3.1.0",
     "hvac>=0.7.0,<0.8.0",
-    "jenkins-job-builder~=4.3.0",
+    "jenkins-job-builder~=6.4.2",
```

Comment:
now that we have renovate to bump version, let's pin exact version in `pyproject.toml`